# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1436505030 14400
#      Fri Jul 10 01:10:30 2015 -0400
# Branch stable
# Node ID 227d582fa3004d7dcee9f82cd6fa085485999817
# Parent  45d7be39198246e8e5cfd42aad05843e266fffa2
build: Sort generated PKG_ADD contents consistently

* libinterp/mk-pkg-add: Set LC_COLLATE=C when sorting for consistent output.

diff --git a/libinterp/mk-pkg-add b/libinterp/mk-pkg-add
--- a/libinterp/mk-pkg-add
+++ b/libinterp/mk-pkg-add
@@ -25,17 +25,17 @@ SED=${SED:-'sed'}
 for f in "$@"; do
   if [ -f $f ]; then
 
     ## Compute and print the autoloads.
 
     base=`basename $f | $SED 's/\.df$//'`
     fcns=`$SED -n -e 's/^ *XDEFUN_DLD_INTERNAL *( *\([^, ]*\)[, ].*$/\1/p' \
                   -e 's/^ *XDEFUNX_DLD_INTERNAL *( *"\([^"]*\)".*$/\1/p' $f | \
-          sort -u`
+          LC_COLLATE=C sort -u`
     if [ -n "$fcns" ]; then
       for n in $fcns; do
         if [ "$n" = "$base" ]; then
           true
         else
           echo "autoload (\"$n\", \"$base.oct\");"
         fi
       done
